pkg/odo/cli/component/create.go

```diff
@@ -1044,33 +1004,6 @@ func (co *CreateOptions) Run() (err error) {
 	return
 }
 
-// The general cpu/memory is used as a fallback when it's set and both min-cpu/memory max-cpu/memory are not set
-// when the only thing specified is the min or max value, we exit the application
-func ensureAndLogProperResourceUsage(resource, resourceMin, resourceMax, resourceName string) {
```

Comment:
These validations might need to happen at push now, as a user can set them using odo config and then push

Reply:
Or better yet when these are set using odo config

Reply:
I added a new commit to add validation for `push` and `config set` command. Would that suffice?

Reply:
yeah